app/controllers/taggings_controller.rb

```diff
@@ -1,14 +1,12 @@
 class TaggingsController < ApplicationController
   before_action :authenticate_user!
-  before_action :set_movie
+  before_action :set_movie, only: [:create, :destroy]
 
   include TmdbHandler
 
   def create
-    @tag_names = params[:tag_list]
     @list = List.find(params[:list_id]) if params[:list_id].present?
```

Comment:
We still need to hang on to `@list` and `@movie` at this point. I'll do a view-focused PR after getting through all of the `tmdb_handler_` methods.